lib/adjust/adjust.performance.js

```diff
@@ -0,0 +1,18 @@
+const { generateArrays, times: { TIME_100K } } = require('../_internal/helpers/performance')
+
+module.exports = function() {
+  const length = TIME_100K[2]
+  const [arr] = generateArrays([length, length, length], x => x)
+  const func = x => x + 10
+
+  const sets = [
+    [func, 0, arr], // first element
+    [func, length / 2, arr], // middle element
+    [func, length - 1, arr] // last element
+  ]
+
+  return {
+    argss: sets,
+    type: 'start_half_end'
```

Comment:
type-first here and make `sets` inline to `argss`

Reply:
done